lib/rubygems/sigstore/openid.rb

```diff
@@ -12,13 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-module Gem
-  module Sigstore
-  end
-end
-
-require "rubygems/sigstore/config"
-require "rubygems/sigstore/crypto"
+require 'rubygems/sigstore/config'
+require 'rubygems/sigstore/crypto'
```

Comment:
If you are fixing the require statements & namespaces in this pr you can probably rename `crypto` to `pkey` to match the class name.
```suggestion
require 'rubygems/sigstore/pkey'
```